### PR TITLE
Daemon network public

### PR DESCRIPTION
**Pull request by Christop1964**
Daemon network documentation update

### CODE REVIEW

File: roles/core/precheck/tasks/prepare.yml [hunk 1]
```diff
@@ -34,14 +34,14 @@
       notify: reload-sshd
 
     # prepare | Update ListenAddress at sshd_config for daemon network
-    - name: Update ListenAddress at /etc/ssh/sshd_config
+    - name: prepare | Restrict SSH access to admin nodename
       lineinfile:
         path: /etc/ssh/sshd_config
         state: present
         regexp: '#ListenAddress 0.'
         line: 'ListenAddress {{ ansible_fqdn }}'
```
Comment (acch) [inline roles/core/precheck/tasks/prepare.yml:42]: As discussed, I guess this should be `scale_admin_nodename`...

**Review by acch — CHANGES_REQUESTED** [review]
Thanks Christoph. Looks good... but I'd suggest to change the address you're restricitng communication for to the `scale_admin_nodename`.

BTW: You can make further changes to this PR by pushing more commits to your branch.

File: roles/core/precheck/tasks/prepare.yml (hunk 1 above)
Comment (Christop1964) [reply roles/core/precheck/tasks/prepare.yml:42]: Achim, 
yes you are right I forgot to change this. 
I will change it

**Comment by Christop1964** [comment]
Closing pull request because of unnecessary files.
Creating a new one.

**Comment by Christop1964** [comment]
Closing pull request because of unnecessary files.
Creating a new one.

**Pull request closed without merging**